pybamm/models/full_battery_models/lithium_ion/electrode_soh_half_cell.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, name="ElectrodeSOH model"):
 
         x_100 = pybamm.Variable("x_100", bounds=(0, 1))
         x_0 = pybamm.Variable("x_0", bounds=(0, 1))
-        Q_w = pybamm.InputParameter("Q_w")
+        Q_w = pybamm.Parameter("Total lithium in the working electrode [mol]")
```

Comment:
IMO better to keep this as the input parameter

Reply:
Really I think these are all use-cases for PsuedoParameters